lms/djangoapps/bulk_email/models.py

```diff
@@ -127,7 +156,7 @@ def ensure_valid_cohort(cls, cohort_name, course_id):
         if cohort_name is None:
             raise ValueError("Cannot create a CohortTarget without specifying a cohort_name.")
         try:
-            cohort = CourseUserGroup.get(name=cohort_name, course_id=course_id)
+            cohort = CourseUserGroup.objects.get(name=cohort_name, course_id=course_id)
```

Comment:
That should be fine.  The number of cohorts per class should always be low enough that a scan won't slow things down.

Load testing, will of course help us be sure.